lib/ansible/module_utils/openstack.py

```diff
@@ -117,6 +117,11 @@ def openstack_cloud_from_module(module, min_version='0.12.0'):
     except ImportError:
         module.fail_json(msg='openstacksdk is required for this module')
 
+    if min_version:
+        min_version = max(StrictVersion('0.12.0'), StrictVersion(min_version))
+    else:
+        min_version = StrictVersion('0.12.0')
```

Comment:
This has broken OpenStack ansible modules in 2.8.8. min_version should be a string, not a StrictVersion. We get the following error:

TypeError: expected string or buffer

This affects kolla-ansible, and probably a few other things too. https://bugs.launchpad.net/kolla-ansible/+bug/1866181